include/cute/atom/copy_traits_xe.hpp

```diff
@@ -67,286 +67,419 @@ auto get_coordinates(cute::Stride<IntT, Int<1>, IntT> ,
+    // Map from (src-thr,src-val) to bit
+    using SrcLayout = Layout<Shape<_16, Shape<_16, _32>>,
+            Stride<_32, Stride<_512, _1>>>;
+    // Map from (dst-thr,dst-val) to bit
+    using DstLayout = Layout<Shape<_16, Shape<_16, _32>>,
+            Stride<_32, Stride<_512, _1>>>;
+    // Reference map from (thr,val) to bit
+    using RefLayout = DstLayout;
+    using CopyInternalType = ushort;
+};
+
+template <class GTensor>
+struct Copy_Traits<XE_2D_U16x16x16x1x1_LD_N::PREFETCH, GTensor>
+    : XE_2D_PF_Unpack<XE_2D_U16x16x16x1x1_LD_N::PREFETCH, GTensor> {
+
+    template <class... CopyArgs>
+    CUTE_HOST_DEVICE Copy_Traits(Copy_Traits<CopyArgs...> const &traits)
+        : XE_2D_PF_Unpack<XE_2D_U16x16x16x1x1_LD_N::PREFETCH, GTensor>(
+                  traits.tensor) {}
+
+    // Logical thread id to thread idx
+    using ThrID = Layout<_16>;
+    // Map from (src-thr,src-val) to bit
+    using SrcLayout = Layout<Shape<_16, Shape<_16, _16>>,
+            Stride<_16, Stride<_256, _1>>>;
+    // Map from (dst-thr,dst-val) to bit
+    using DstLayout = Layout<Shape<_16, Shape<_16, _16>>,
+            Stride<_16, Stride<_256, _1>>>;
+    // Reference map from (thr,val) to bit
+    using RefLayout = DstLayout;
+    using CopyInternalType = ushort;
 };
 
 template <class GTensor>
 struct Copy_Traits<XE_2D_U32x8x16x1x1_LD_N, GTensor>
-     : XE_2D_LD_Unpack<XE_2D_U32x8x16x1x1_LD_N, GTensor>
-{
-  // Logical thread id to thread idx
-  using ThrID = Layout<_16>;
-  // Map from (src-thr,src-val) to bit
-  using SrcLayout = Layout<Shape<_16, _64>, Stride<_0, _1>>;
-  // Map from (dst-thr,dst-val) to bit
-  using DstLayout =
-    Layout<Shape<_16, Shape<_8, _32>>, Stride<_32, Stride<_512, _1>>>;
-  // Reference map from (thr,val) to bit
-  using RefLayout = DstLayout;
-  using CopyInternalType = uint;
+    : XE_2D_LD_Unpack<XE_2D_U32x8x16x1x1_LD_N, GTensor> {
+    // Logical thread id to thread idx
+    using ThrID = Layout<_16>;
+    // Map from (src-thr,src-val) to bit
+    using SrcLayout = Layout<Shape<_16, _64>, Stride<_0, _1>>;
+    // Map from (dst-thr,dst-val) to bit
+    using DstLayout
+            = Layout<Shape<_16, Shape<_8, _32>>, Stride<_32, Stride<_512, _1>>>;
+    // Reference map from (thr,val) to bit
+    using RefLayout = DstLayout;
+    using CopyInternalType = uint;
 };
 
 template <class GTensor>
 struct Copy_Traits<XE_2D_U16x8x16x4x2_LD_N, GTensor>
-     : XE_2D_LD_Unpack<XE_2D_U16x8x16x4x2_LD_N, GTensor>
-{
-  // Logical thread id to thread idx
-  using ThrID = Layout<_1>;
-  // Map from (src-thr,src-val) to bit
-  using SrcLayout = Layout<Shape<_1, Shape<_1, _1>>>; // one coordinate
-  // Map from (dst-thr,dst-val) to bit
-  using DstLayout = Layout<Shape<_1, Shape<_64, _1>>>;
-  // Reference map from (thr,val) to bit
-  using RefLayout = SrcLayout;
-  using CopyInternalType = ushort;
+    : XE_2D_LD_Unpack<XE_2D_U16x8x16x4x2_LD_N, GTensor> {
+    // Logical thread id to thread idx
+    using ThrID = Layout<_16>;
```

Comment:
Is there a reason for changing this to `Layout<_16>` instead of keeping it `Layout<_1>`?